src/fontra/views/editor/scene-controller.js

```diff
@@ -1462,3 +1524,66 @@ function splitLocation(location, glyphAxes) {
 
   return { fontLocation, glyphLocation };
 }
+
+export function connectTwoDistinctContours(path, firstPointIndex, secondPointIndex) {
+  let selectedPointIndices = [];
+
+  const [firstContourIndex, firstContourPointIndex] =
+    path.getContourAndPointIndex(firstPointIndex);
+  const [secondContourIndex, secondContourPointIndex] =
+    path.getContourAndPointIndex(secondPointIndex);
+
```

Comment:
Maybe we need some sanity checking here:
- assert that the contour indices are indeed different
- assert that .isStartOrEndPoint() is true for both indices
- assert that firstPointIndex < secondPointIndex

Throw an exception if any of the above does not hold.

Reply:
The first two items are covered in `getSelectedJoinContoursPointIndices`, therefore we know what we pass into `joinContours` is valid. Let me add the last item to with a sorting to getSelectedJoinContoursPointIndices and we are done. Or do you want to be on the save side if we may want to use joinContours somewhere else in future?

Reply:
OK, done. Except the last point I made the point indices swap. I hope you find it also a valid solution – if not, I'll change it.

Reply:
OK, sorry. Changed it as requested.